module/motion/HeadController/data/config/nugus1/HeadController.yaml

```diff
@@ -1,3 +1,8 @@
+log_level: WARN
```

Comment:
I can't see any difference in the head controller nugus1 and nugus2 and default config?